tests/integration/worker/test_worker_util.py

```diff
@@ -0,0 +1,127 @@
+import mock
+
+from rest_framework.test import APITestCase
+from scripts.workers.worker_util import EvalAI_Interface, URLS
+
+
+AUTH_TOKEN = "mock_auth_token123456"
```

Comment:
we can define these variables in setUp itself.

Reply:
Done!! @pushkalkatara